media-gfx/drawpile/drawpile-2.0.10.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="A collaborative drawing program"
+DESCRIPTION="Drawpile is a drawing program that lets you share\
+ the canvas with other users in real time."
+HOMEPAGE="http://drawpile.net"
+COPYRIGHT="2018 Calle Laakkonen"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/drawpile/Drawpile/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="0ff79e9f95ca02ff016eed3118a83ca62e80a1ca93fe11e3fb8ecd22c1fe37a9"
+SOURCE_DIR="Drawpile-$portVersion"
+PATCHES=""
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	drawpile$secondaryArchSuffix = $portVersion
+	app:Drawpile = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libkf5archive$secondaryArchSuffix
+	lib:libminiupnpc$secondaryArchSuffix
+	lib:libsodium$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libkf5archive$secondaryArchSuffix
+	devel:libgif$secondaryArchSuffix
+	devel:libminiupnpc$secondaryArchSuffix
+	devel:libsodium$secondaryArchSuffix
+	devel:libarchive$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:qmake$secondaryArchSuffix
```

Comment:
if it is cmake based, then it shouldn't need qmake. please check it, and remove it if unneeded.